### PR TITLE
Introducing a document describing the event detection process (PPTT-3…

### PR DESCRIPTION
**Pull request by wronan**
…647)

### CODE REVIEW

**Comment by JacekClearcode** [comment]
What version should this go to? `master` is not a valid option ;)

**Comment by wronan** [comment]
Sorry - I'll never learn...

File: data_collection/processing/event_type_detection.rst [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+~~~~~~~~~~~~~~~~~~
+
+Then the `ping` parameter is analyzed. Depending on it's value one of the following ping types will be assigned:
+
+- values `1`, `2` and `3` result in `Heartbeat` ping type
+- value of `4` results in `Deanonymization` ping type
+- value of `5` results in `PagePerformance` ping type
+- value of `6` results in `CustomPing` ping type
+
+Values outside of that range will cause an error which results in broken event.
```
Comment (adaqus) [inline data_collection/processing/event_type_detection.rst:54]:
```suggestion
Values outside of that range will cause an error which results in BrokenEvent.
```

File: data_collection/processing/event_type_detection.rst [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+Abandoned Cart
+~~~~~~~~~~~~~~~~~~
+
+When a session did not track a `OrderCompleted` event, the last event of that type will be converted to `AbandonedCart`.
+
+Excluded Event
+~~~~~~~~~~~~~~~~~~
+
+There are several ways of excluding an event (e.g. by blacklisting source IP or User-Agent header matching).
+If an event matches given criteria it will be excluded from the reports but is still tracked and receives `ExcludedEvent` type.
```
Comment (adaqus) [inline data_collection/processing/event_type_detection.rst:144]: It is also worth to mention that ExcludedEvents are visible in Tracker Debugger.

**Review by adaqus — APPROVED** [review]

File: data_collection/processing/event_type_detection.rst [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+First step in the event type detection process is a check for conversion signs in the tracked event.
+If your tracked event had an `idgoal` parameter with the ID value other than `0` then it will be marked as `GoalConversion` type.
+
+Ping
+~~~~~~~~~~~~~~~~~~
+
+Then the `ping` parameter is analyzed. Depending on it's value one of the following ping types will be assigned:
+
+- values `1`, `2` and `3` result in `Heartbeat` ping type
```
Comment (RomanSek) [inline data_collection/processing/event_type_detection.rst:49]: Looks good, but quotes using single "`" character in RST format are formatted as _italic_, if you want to use `code` format they should be double "``" character quotes.

Example:
```
... result in ``Heartbeat`` ping type
```

Here's a [preview](https://developers.piwik.pro/en/pptt-3647_change_event_detection_order/data_collection/processing/event_type_detection.html) of this branch.

**Review by RomanSek — APPROVED** [review]

File: data_collection/processing/event_type_detection.rst (hunk 3 above)
Comment (wronan) [reply data_collection/processing/event_type_detection.rst]: I was aiming at any kind of emphasis, italics looked good enough but indeed code styling looks better. I just thought that there's something wrong with the formatting and the code block just looks like a italic block :)
Thx for the tip.

**Pull request merged**